src/Connector.js

```diff
@@ -71,6 +71,11 @@ class Connector {
     await this.client.unregister(id);
   }
 
+  async listDevices() {
+    const { devices } = await this.client.getDevices();
+    return devices;
+  }
```

Comment:
The code LGTM, I just didn't understand the need of adding the `listDevices` operation :confused:

Reply:
Actually, the `listDevices` operation shouldn't be added to this PR. It will be necessary to guarantee the connector won't send unneeded `updateSchema` and `updateConfig` operations. The way API is designed today we would end up with an infinite looping if things state not checked.

fog -> device.config.updated -> connector -> device.config.sent -> cloud
fog <- device.config.sent <- connector <- device.config.updated <- cloud

Besides fixing this issue, coming back with the `DeviceStore` in the connector service will be fundamental to improve performance and increase consistency.

Reply:
I will remove it from this patch but let me know if you understood the problem.

Reply:
Commit removed.